nvidia/gmsl_camera/frame_sync.rst

```diff
@@ -0,0 +1,189 @@
+.. _frame_sync:
+
+Frame Synchronization 
+#####################
+
+RQX-58G supports **Free Run Mode** and **Frame Sync Mode**.
+
+Typically, if you want to use FSYNC in RQX-58G, you need to follow the procedure.
+    
+    i.   Set all cameras to frame sync mode by kernel modules's parameter.
+    ii.  Use I/O library to trigger frames, e.g. Neuron Library.
+    iii. Use Argus API or GStreamer to retrive your frame.
+
+| Free run mode means if you are expecting every camera will have 30 FPS frame rate. 
+| But every camera has different clock source, so they will not able to output the frame at the same time.
+
+| Like the following plot.
+| Camera's frame 1 (f1) and camera's frame 2 (f2) is not at the same time, therefore, under your system they are not under the same timestamp.
+
+.. image:: images/free-run-30fps.png
+  :width: 80%
+  :align: center
+
+| Frame Sync mode is a mechanism to make sure all of your camera is in passive mode, and every frame can by triggered by external signal. 
```

Comment:
all of your camera"s" "are" in passive mode,

Reply:
Done